### DISCUSSION TITLE
Create repository using my electron app.

### DISCUSSION THREAD

**Discussion by CoffeWood** [community/community#20329] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, My name is CoffeWood and I am new user of GitHub.
I am working now on simple desktop app using electron, witch will work with my profile and creating repository.
Thank you for your help.

**Comment by Tostisto** [comment, marked as answer]
Hi, GitHub are providing [GitHub rest API](https://docs.github.com/en/rest) for projects like your. If you just want create repository you will probably use this [page](https://docs.github.com/en/rest/repos/repos)

**Reply by CoffeWood** [reply]
Thank you, can I contact you with some questions?

**Reply by Tostisto** [reply]
You are welcome
